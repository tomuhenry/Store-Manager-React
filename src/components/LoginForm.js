import React from 'react';

const LoginForm = props => {
  return (
    <div className='row justify-content-center flex-container'>
    <meta name="viewport" content="width=device-width, initial-scale=1"></meta>
      <div className='flex-login'>
      <h1>Login Here</h1>
        <form onSubmit={props.FormSubmit} id='login'>
          <input
            type='email'
            placeholder='Email...'
            id='email'
            name='email'
            onChange={props.changed}
          />
          <input
            type='password'
            placeholder='Password...'
            id='password'
            name='password'
            onChange={props.changed}
          />
          <input type='submit' value="Login" className='form_button' />
          <span id="message"/><br/>
          <a href='' id='forgot-password-link'>
            Forgot password?
          </a>
        </form>
      </div>
    </div>
  );
};

export default LoginForm;
