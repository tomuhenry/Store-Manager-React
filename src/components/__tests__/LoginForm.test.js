import React from 'react';
import { shallow } from 'enzyme';
import LoginForm from '../LoginForm';

describe('App', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<LoginForm />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should check for div', () => {
    const wrapper = shallow(<LoginForm />);
    expect(wrapper.find('div').length).toBeGreaterThanOrEqual(1);
  });
});
