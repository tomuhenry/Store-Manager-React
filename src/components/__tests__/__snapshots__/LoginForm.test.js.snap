// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="row justify-content-center flex-container"
>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <div
    className="flex-login"
  >
    <h1>
      Login Here
    </h1>
    <form
      id="login"
    >
      <input
        id="email"
        name="email"
        placeholder="Email..."
        type="email"
      />
      <input
        id="password"
        name="password"
        placeholder="Password..."
        type="password"
      />
      <input
        className="form_button"
        type="submit"
        value="Login"
      />
      <span
        id="message"
      />
      <br />
      <a
        href=""
        id="forgot-password-link"
      >
        Forgot password?
      </a>
    </form>
  </div>
</div>
`;
